app/packages/core/src/components/Grid/Grid.tsx

```diff
@@ -49,20 +47,39 @@ function Grid() {
   const setSample = fos.useExpandSample(store);
   const getFontSize = useFontSize(id);
 
+  const lookerStore = useMemo(() => {
+    /** LOOKER STORE REFRESHER */
+    reset;
+    /** LOOKER STORE REFRESHER */
+
+    return new LRUCache<string, Lookers>({
```

Comment:
Moving this back to LRU cache, so we can explicitly delete lookers and keep memory down